src/aquto.js

```diff
@@ -89,22 +80,8 @@ function genericCheckEligibility(options) {
  */
 function checkOfferWallEligibility(options) {
   var data = { apiVersion: 'v8' }
-  if(options.carrier) {
-    data.operatorCode = options.carrier
-  }
-  if(options.phoneNumber) {
-    data.phoneNumber = options.phoneNumber
-  }
-  if(options.countryCode) {
-    data.countryCode = options.countryCode
-  }
-  if(options.publisherSiteUuid) {
-    data.publisherSiteUuid = options.publisherSiteUuid
-  }
-  if(options.channel) {
-    data.publisherSiteUuid = options.channel
-  }
-
+  utils.assignFields(data)
+  utils.applyParams(data, options)
```

Comment:
No need to have two separate functions. Just use one e.g. applyParams. Also, what's this "utils" object? Should just be calling applyParams directly.